### ISSUE TITLE
Per-Key RGB

### ISSUE THREAD

**Issue by fscottcopeland** [seerge/g-helper#3482]
### Rules

- [X] I made myself familiar with the <a href='https://github.com/seerge/g-helper?tab=readme-ov-file'>Readme</a>, <a href='https://github.com/seerge/g-helper/wiki/FAQ'>FAQ</a> and <a href='https://github.com/seerge/g-helper/wiki/Troubleshooting'>Troubleshooting</a>.
- [X] I understand that, if insufficient information will be provided, my issue will be closed without an answer.

### Is your feature request related to a problem? Please describe

Is there any possibility of implementing per-key RGB?

### Describe the solution you'd like

As I have all ASUS programs uninstalled, I no longer have the ability to have a Per-Key RGB setting. It would be great to have this.

If you need exports from my laptop (Duo 16 GX650) to enable this, I'm very happy to help!

### Device and Model

Duo 16 GX650

**Comment by seerge** [comment]
@fscottcopeland hello, what exactly do you understand under per-key RGB ? way to set static colors for each keys ?

Cause in general, app already uses per-key RGB settings for Ambient mode for example (to color sections of the keyboard). But since I don't have a device with per-key RGB it's very difficult to test or develop anything for it :)

**Comment by fscottcopeland** [comment]
Yeah, as in the ability to assign specific colours to specific keys. You could do this with Aura Creator (random screenshot below)
![image](https://github.com/user-attachments/assets/4dfc8c68-a653-467a-99a5-f84b26542829)
My laptop supports per-key rgb so it'd be a great feature to have. Perhaps even with a method to have a different preset depending on the power mode selected.

**Comment by fscottcopeland** [comment]
@seerge Let me know if there's any logs, exports etc. I can provide to assist with this :)

**Comment by seerge** [comment]
@fscottcopeland on this stage it's not much of a logs, but rather a device itself that I need :) cause I need to test things in action.

This is not some single new ON/OFF toggle that is easy to add based on logs or user reports only ...

**Comment by chleved** [comment]
@fscottcopeland, if you are willing to have the asus suite bloat on your pc you can have the best of all worlds. On my pc i have installed Armoury Crate, Aura Creator and G-Helper. Armoury crate is just there so i can bind the M4/Fn+F5 to "play next song" and i have a custom profile on Aura Creator with custom effects on press (OpenRGB doesn't see some of the rgbs on my laptop). I use G-Helper for everything else since it is objectively better. Just remember to stop the asus services in g-helper with the appropriate button. Also some times when changing mode, the rgbs might go back to g-helper profile, to fix that you just restart "ASUS AURA SYNC lighting service". I made a task in task scheduler where that service gets restarted at every boot.
